staff_features/resources/resource_merge.feature

```diff
@@ -1,35 +1,34 @@
-    Given the Resource 'A' is opened in edit mode
+      And two Resources have been created
+  Scenario: Merge two Resources by browsing
+    Given the first Resource is opened in edit mode
      When the user clicks on 'Merge'
-      And the user clicks on 'Browse' in the dropdown menu
-      And the user selects Resource 'B'
-      And the user clicks on 'Link'
-      And the user clicks on 'Merge' in the confirm popup
-     Then the 'Resource' merged message is displayed
-      And the data from Resource 'B' is added to the Resource 'A'
-      And the Resource 'B' is deleted
-  Scenario: Merge two Resources by searching the Resource
-    Given the Resource 'A' is opened in edit mode
+      And the user clicks on the dropdown in the merge dropdown form
+      And the user clicks on 'Browse' in the merge dropdown form
+      And the user filters by text with the second Resource title in the modal
+      And the user selects the second Resource from the search results in the modal
+      And the user clicks on 'Link' in the modal
+      And the user clicks on 'Merge' in the merge dropdown form
+      And the user clicks on 'Merge' in the modal
+     Then the 'Resource(s)' merged message is displayed
+      And the second Resource is deleted
+      And the following linked records from the second Resource are appended to the first Resource
+        | Related Accessions |
+        | Agent Links        |
+        | Subjects           |
+        | Classifications    |
+  Scenario: Merge two Resources by searching
+    Given the first Resource is opened in edit mode
      When the user clicks on 'Merge'
-      And the user fills in 'Resource' with 'B'
-      And the user selects 'B' from the suggestions
-      And the user clicks on 'Merge'
-      And the user clicks on 'Merge' in the confirm popup
-     Then the 'Resource' merged message is displayed
-      And the data from Resource 'B' is added to the Resource 'A'
-      And the Resource 'B' is deleted
-    Scenario: Cancel Merge Before Confirmation
-      Given the Resource 'A' is opened in edit mode
-       When the user clicks on 'Merge'
-        And the user fills in 'Resource' with 'B'
-        And the user selects 'B' from the suggestions
-        And the user clicks on 'Merge'
-        And the user clicks on 'Cancel' in the confirm popup
-       Then the confirm popup is disappeared
-        And the Resources are not changed
-      
\ No newline at end of file
+      And the user fills in and selects the second Resource in the merge dropdown form
+      And the user clicks on 'Merge' in the merge dropdown form
+      And the user clicks on 'Merge' in the modal
+     Then the 'Resource(s)' merged message is displayed
+      And the second Resource is deleted
+      And the following linked records from the second Resource are appended to the first Resource
+        | Related Accessions |
+        | Agent Links        |
+        | Subjects           |
+        | Classifications    |
```

Comment:
I would also add the Cancel Merge before confirmation scenario as this is testing an edge case and improves feature robustness.

Scenario: Cancel Merge Before Confirmation
Given the Resource 'A' is opened in edit mode
When the user clicks on 'Merge'
And the user fills in and selects the 'B' Resource in the merge dropdown form
And the user clicks on 'Merge' in the merge dropdown form
And the user clicks on 'Cancel' in the modal
Then the confirmation modal is dismissed
And Resource 'A' and Resource 'B' remain unchanged